elastic/metadata.csv

```diff
@@ -72,7 +72,7 @@ elasticsearch.indexing.index.total,gauge,,document,,The total number of document
 elasticsearch.indexing.index.total.count,count,,document,,The total number of documents indexed to an index.,0,elasticsearch,total indexed,
 elasticsearch.indices.count,gauge,,index,,The number of indices in the cluster.,0,elasticsearch,number of indices,
 elasticsearch.indices.indexing.index_failed,gauge,,,,The number of failed indexing operations [v2.1+].,-1,elasticsearch,index fails,
-elasticsearch.indices.indexing.index_failed.count,count,,,,The number of failed indexing operations [v2.1+].,-1,elasticsearch,index fails,
+elasticsearch.indices.indexing.index_failed.count,count,,,,The number of failed indexing operations submitted as a count [v2.1+].,-1,elasticsearch,index fails count,
```

Comment:
The `submitted as a count` clarification seems to be missing in many similar cases: https://github.com/DataDog/integrations-core/blob/683058908ef9717387f29ccdfe565f98896d36e3/elastic/metadata.csv#L306

Reply:
Hm yeah looks like I will need to fix this for many other metrics. I'll update the PR name since the scope has changed to fix for all metrics.